automated/roles/awx_on_k3s/tasks/deploy_k3s.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: deploy_k3s | Install k3s
+  ansible.builtin.shell: set -o pipefail && curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=v1.28.7+k3s1 sh -s - --write-kubeconfig-mode 644 {% if awx_on_k3s_local_hosts is defined %}--resolv-conf /etc/rancher/k3s/resolv.conf{% endif %}
```

Comment:
The K3S_version should be a variable to cater for future versions

Reply:
yea this was just an initial throw together in a few hours. I was also considering using a different role to deploy it that other people maintain and have more functionality when I started writing this. But after getting it to work and thinking about it more I think it would likely cause a lot of issues so I will make this a variable

Reply:
F.Y.I., there is a official playbook to deploy K3s: https://github.com/k3s-io/k3s-ansible